lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100vw;
```

Comment:
I think this is leftover from mobile, but works the same for this purpose. do you prefer just 100%?